.bazelci/presubmit.yml

```diff
@@ -857,12 +865,6 @@ tasks:
     working_directory: examples/bzlmod/proto
     build_targets:
       - "//..."
-  bzlmod_proto:
-    name: Proto and Prost with prebuilt protoc with bzlmod
-    platform: ubuntu2004
-    working_directory: examples/bzlmod/proto_with_toolchain
-    build_targets:
-      - "//..."
```

Comment:
is this removal intentional?

Reply:
Nope, that was an accident. Probably a cut instead of copy. 
 Good catch. Thanks for point it out.